merlin/systems/dag/runtimes/nvtabular/executor.py

```diff
@@ -0,0 +1,305 @@
+#
+# Copyright (c) 2023, NVIDIA CORPORATION.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+import functools
+import itertools
+import logging
+
+from merlin.core.compat import cudf
+from merlin.core.compat import cupy as cp
+from merlin.core.compat import numpy as np
+from merlin.core.compat import pandas
+from merlin.core.dispatch import build_cudf_list_column, concat_columns, is_list_dtype
+from merlin.dag import Graph, Node, Supports
+from merlin.dag.executors import LocalExecutor
+from merlin.table import CupyColumn, NumpyColumn, TensorTable
+
+LOG = logging.getLogger("merlin-systems")
+
+
+class NVTabularServingExecutor(LocalExecutor):
```

Comment:
Is the idea that we're working toward unifying the functionality required here into `LocalExecutor` entirely?

Reply:
As much as possible, yes. The ideal end state would be getting rid of this executor entirely 👍🏻